pylint/checkers/exceptions.py

```diff
@@ -225,14 +245,28 @@ def visit_instance(self, instance: objects.ExceptionInstance) -> None:
     def visit_classdef(self, node: nodes.ClassDef) -> None:
         if not utils.inherit_from_std_ex(node) and utils.has_known_bases(node):
             if node.newstyle:
-                self._checker.add_message("raising-non-exception", node=self._node)
+                self._checker.add_message(
+                    "raising-non-exception",
+                    node=self._node,
+                    confidence=INFERENCE,
+                )
 
     def visit_tuple(self, _: nodes.Tuple) -> None:
-        self._checker.add_message("raising-bad-type", node=self._node, args="tuple")
+        self._checker.add_message(
+            "raising-bad-type",
+            node=self._node,
+            args="tuple",
+            confidence=INFERENCE,
```

Comment:
Why inference and not high? Also it should be done in another MR.

Reply:
It's in a sep commit but np, I can split that out. Also, I'm pretty sure I traced all the code / funcs that received inference results, but I could of course have made a mistake. Can move it to a sep PR for discussion.

Reply:
Thank you !